pipeline/load/display_metrics.py

```diff
@@ -1,59 +1,34 @@
-import operator
+from typing import Dict, List
 
 import prefect
 from prefect import task
 
-from pipeline.types import MatchingMetrics, MatchingOutput, User
+from pipeline.types import MatchingMetrics, MatchingOutput, User, UserId
+
+
+def render_counts_per_user(
+    users: List[User], counts: Dict[UserId, int]
+) -> List[str]:
+    """Helper method to get lines of counts per user in descending order."""
+    display_names = {user.uid: user.displayName for user in users}
+    # Sort entries by count in descending order
+    sorted_items = sorted(counts.items(), key=lambda p: p[1], reverse=True)
```

Comment:
We don't need to use the `operator.itemgetter` module to sort by counts, we can use a lambda and sort by the second item in the tuple. I would prefer to avoid adding a dependency, but either way, I add a comment to clearly explain what is happening, because the `1` and the `reverse=True` is hard to remember the meaning of.

Reply:
Very cool! I learned something new today. I didn't know [itemgetter is faster](https://stackoverflow.com/a/58171096).

I decided to take the dependency anyway and use itemgetter. It's not a big deal performance-wise, but I like your idea and want to use it as an example.